tests/e2e/pages/AccountPageSFRA.js

```diff
@@ -29,6 +29,7 @@ export default class AccountPage {
 
     addCard = async (cardData) => {
         await t
+            .expect(Selector('.card-error').visible).notOk()
```

Comment:
I see the use case for this now but in the future we will probably forget why we have an expect statement in a helper function and it will look weird